Web/Admin.UI/src/app/components/account/account-config-form/account-config-form.component.ts

```
@@ -0,0 +1,182 @@
+    user.roles = this.rolesControl.value;
+    user.username = this.firstName.value + '.' + this.lastName.value;
+    if (this.accountForm.status == 'VALID') {
+      if (this.formMode == FormMode.Create) {
+        this.accountService.createUser(user).subscribe({
+          next: (response) => {
+            if (response) {
+              this.submittedConfiguration.emit({success: true, message: ""});
+            }
+          },
+          error: (err) => {
+            if (err.status = 409) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)